configs/votenet/votenet_iouloss_8x8_scannet-3d-18class.py

```diff
@@ -0,0 +1,45 @@
+_base_ = [
```

Comment:
We may directly inherit this config from `votenet_8x8_scannet-3d-18class.py` thus the config only contains the modification by iouloss.